tests/modules/audit_logs/resources/test_audit_log.py

```diff
@@ -67,12 +67,13 @@ def test_audit_asset_group_creation(
             flask_app_client, admin_user, sighting_uuid
         ).json
 
-        assert len(sighting_audit) == 2
-        create_entry = sighting_audit[1]
+        # TODO why does this result in an audit for the AGS process identification being recreated?
```

Comment:
@bluemellophone This looks suspicious. This generates an audit log anf warning as anon user is trying to recreate the progress identification on the AGS

Reply:
This is suspicious.  I need to slightly re-work the initialization of the various progress trackers.  Overwriting a progress tracker is not, in of itself, an issue however.  For example, we may chose to re-run detection and discard the previous tracker.

Reply:
In fact, these aren't really faults, mostly just warnings that we would want to know happened.  I do not really consider this an indicator of an error outright, for example.

Reply:
Agreed, now downgraded to audit log warning.
`{'audit_type': 'Object Warning',
  'created': '2022-06-06T09:22:52.109413+00:00',
  'duration': None,
  'elasticsearchable': True,
  'guid': '41f41080-9132-44b4-aaea-f62a54d8d5aa',
  'indexed': '2022-06-06T09:22:52.109442+00:00',
  'item_guid': '3c973271-c798-4d4b-8415-b88a353fd96a',
  'message': 'Sighting <Sighting(guid=3c973271-c798-4d4b-8415-b88a353fd96a, '
             'stage=identification, )> already has a progress identification '
             '<Progress(guid=4d01e14a-947b-477a-a01f-5d4a323575a9, '
             "description='Sage identification for Sighting "
             "UUID('3c973271-c798-4d4b-8415-b88a353fd96a')', status=created, "
             'percentage=0, eta=None, active=True, complete=False, '
             'skipped=False, failed=False, skipped=False, created=2022-06-06 '
             '09:22:52.022355, updated=2022-06-06 09:22:52.080849)> executed '
             'by researcher1@localhost(333c7413-4afd-45ee-a730-74e2ac6a9e89)',
  'module_name': 'Sighting',
  'user_email': 'researcher1@localhost'},
`
See that the type is now Object Warning as opposed to fault.